src/qdrant/hello.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+from urllib.parse import urljoin
+from httpx import Client, Response
+import os
+from json import dumps
+
+
+def log_request(request):
+    print(f"> {request.method} {request.url}")
+
+
+def log_response(response):
+    request = response.request
+    print(f"< {request.method} {request.url} - {response.status_code}")
+    if response.status_code >= 299:
+        response.read()
+        print(f"\n{response.text}")
+
+
+def main():
+
+    endpoint = os.environ["ENDPOINT"]
+    api_key = os.getenv("API_KEY")
```

Comment:
This API key is required, so should be `os.environ` otherwise it will not raise an error. In the weaviate example the API key is optional for public databases.

Reply:
thanks for catching this.